playback/src/player.rs

```diff
@@ -378,7 +378,7 @@ enum PlayerPreload {
     },
 }
 
-type Decoder = VorbisDecoder<Subfile<AudioDecrypt<AudioFile>>>;
+type Decoder = Box<dyn AudioDecoder + Send>;
```

Comment:
There were already previously two different `VorbisDecoder`s, and it was solved with feature flags. Now, a third decoder is added, and a trait is used instead. It would be more consistent IMHO to use either feature flags or traits for all decoders. A third option (with less overhead than boxed traits) would be an `enum`, which is sometimes an alternative to traits if there's a well-known number of implementors.

Reply:
Feature flags don't work because we need to switch between the two based on runtime flags.
I don't really see a usecase for switching between libogg and lewton at runtime, so feature flags is fine there.

I agree there is a bit of inconsistency, but it seems reasonable and actually simpler this way.

I think the overhead of boxed trait is completely fine in this context.